source/tutorials/openmm.rst

```diff
@@ -17,4 +23,128 @@ folder of the GitHub repository. It contains:
   NanoVer simulation using a python client.
 * `openmm_neuraminidase`: A notebook that demonstrates how to set up an interactive
   OpenMM simulation from scratch with NanoVer and change the visualisations of
-  different atom selections for a protein-ligand system.
\ No newline at end of file
+  different atom selections for a protein-ligand system.
+
+XML Files
+==============================================
+
+The XML format for serializing OpenMM simulations consists of a root element ``<OpenMMSimulation>`` containing three main components:
+
+1. **Starting Structure**: Enclosed in either ``<pdbx>`` or ``<pdb>`` tags
+2. **OpenMM Serialized System**: Enclosed in the ``<System>`` tag
+3. **OpenMM Serialized Integrator**: Enclosed in the ``<Integrator>`` tag
+
+An optional fourth component, the OpenMM serialized state, may also be included.
+The format allows for efficient storage and exchange of OpenMM simulation data, including the starting structure, system configuration, and integrator settings.
+
+XML Structure
+-------------
+
+.. code-block:: xml
+
+    <OpenMMSimulation>
+        <pdbx>
+            <!-- Content of the PDBx file -->
+        </pdbx>
+        <System ...>
+            <!-- XML content of the OpenMM serialized system -->
+        </System>
+        <Integrator ...>
+            <!-- XML content of the OpenMM serialized integrator -->
+        </Integrator>
+    </OpenMMSimulation>
+
+Components Description
+----------------------
+
+1. **Starting Structure**:
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+   * Tag: ``<pdbx>`` (preferred) or ``<pdb>`` (for backward compatibility)
+   * Content: The entire PDBx or PDB file content
+
+2. **OpenMM Serialized System**:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Within this tag, you'll find:
+
+    **Particles:** They are represented with ``<Particle>`` tags, each containing:
```

Comment:
This line is all in bold (default for a nested list in Sphinx). Adding an extra line break stops this (https://stackoverflow.com/questions/42297839/parent-element-of-nested-list-is-bold)